### PR TITLE
Address conditional access related prototype comment

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by AlekseyTs** [comment]
>                 // PROTOTYPE(dataflow): It looks like there is a bug in IOperation tree around XmlMemberAccessExpressionSyntax,

We should probably link this comment to the same issue

---
Refers to: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs:2481 in 429e5ca. [](commit_id = 429e5ca0e027f88e8f4db7ecaea1cf147ac8e9b1, deletion_comment = False)

**Comment by mavasani** [comment]
>                 // PROTOTYPE(dataflow): It looks like there is a bug in IOperation tree around XmlMemberAccessExpressionSyntax,

Ah yes, sure.

---
In reply to: [395549087](https://github.com/dotnet/roslyn/pull/27579#issuecomment-395549087) [](ancestors = 395549087)

---
Refers to: src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs:2481 in 429e5ca. [](commit_id = 429e5ca0e027f88e8f4db7ecaea1cf147ac8e9b1, deletion_comment = False)

**Review by AlekseyTs — APPROVED** [review]
LGTM (iteration 2)

File: src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb [hunk 1]
```diff
@@ -2048,12 +2048,13 @@ End Module
                 Diagnostic(ERRID.ERR_BadIteratorReturn, "Sub").WithLocation(28, 21),
                 Diagnostic(ERRID.HDN_UnusedImportStatement, "Imports System.Collections.Generic").WithLocation(2, 1))
             comp.VerifyAnalyzerDiagnostics({New MemberReferenceAnalyzer}, Nothing, Nothing, False,
-                Diagnostic(MemberReferenceAnalyzer.HandlerAddedDescriptor.Id, "AddHandler, New Object").WithLocation(26, 24),
```
Comment (333fred) [inline src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb]: Why get rid of the IDs?

Comment (mavasani) [reply src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb]: It seems the original test was hand crafted, and hence had the IDs. Failure text contains the string representation of the IDs, so should be easier to maintain.

---
In reply to: [193881051](https://github.com/dotnet/roslyn/pull/27579#discussion_r193881051) [](ancestors = 193881051)

**Comment by mavasani** [comment]
@dotnet-bot test this please

**Review by 333fred — APPROVED** [review]
LGTM (Iteration 2)

**Pull request merged**